noncompiled/JsonSerializer.js

```diff
@@ -1,4 +1,24 @@
 var JsonSerializer 	= {};
+JsonSerializer.model = function (model, options) {
```

Comment:
Love it, definitely a step in the right direction. Thanks for working out the global variable references.